packages/pages/src/common/src/feature/features.ts

```diff
@@ -93,7 +94,9 @@ export const convertTemplateConfigToFeatureConfig = (
   } else {
     featureConfig = {
       ...featureConfigBase,
-      entityPageSet: {},
+      entityPageSet: config.pageUrlField
+        ? { pageUrlField: config.pageUrlField }
```

Comment:
Check with Spruce if they require not setting the key. If not, we don't need this ternary.

Reply:
The field is actually omitted automatically if `pageUrlField` is `null` even without the ternary, so I removed it.